src/common/components/layouts/pages/inspect/tree/Tree.js

```diff
@@ -131,6 +143,8 @@ function Tree({
     const errorMessages = useMemo(() => {
         switch (profile) {
             case errorProfiles.TAGGED_PDF:
+            case errorProfiles.PDFUA_2:
```

Comment:
The same should be added to the `src/common/components/layouts/pages/inspect/pdfDocument/PdfDocument.js`: `errorMessages`